### PR TITLE
Prepare to  Fix JSTOR

### PR DESCRIPTION
**Pull request by GlazerMann**
updates code to use curl and makes it ready to go

### CODE REVIEW

**Comment by ms609** [comment]
Well, we are a robot!
Do we need to get an API key?  How might we obtain one?

**Comment by GlazerMann** [comment]
I got a UUID from my FireFox.  I highly doubt that it will continue to work, but it points to the possibility of working.

**Comment by GlazerMann** [comment]
I will e-mail JSTOR

File: apiFunctions.php [hunk 1]
```diff
@@ -492,7 +492,26 @@ function expand_by_jstor($template) {
...
+      'header'=>"Accept-language: en\r\n" .
+                "Cookie: UUID=3d1209ba-d7d6-40dc-8f1a-9190c982e0f2\r\n"
+    )
+    );
+  } else {
+   $jstor_opts = array(
+    'http'=>array(
+      'method'=>"GET",
+      'header'=>"Accept-language: en\r\n" .
+                "Cookie: UUID=2c114fa7-67a8-4291-a967-e4c5c5a563ac\r\n"
```
Comment (ms609) [inline apiFunctions.php]: This cookie will only exist on your local machine.  CURL has cookie-handling facilities.  But the appropriate thing here is to use JSTOR's API; if this route is not designed for programmatic access, then we should respect JSTOR's wishes and not overload their servers.

Comment (GlazerMann) [reply apiFunctions.php]: I have an email into JSTOR.  @ms609 we can commit as-is for now which JSTOR on the toolforge turned off.

**Comment by GlazerMann** [comment]
CURL does not seem to work...

**Pull request closed without merging**